doc/policy_tutorial/DEV_ENV_SETUP.md

```diff
@@ -0,0 +1,94 @@
+# Setting up the development environment
+
+As was clear from the introduction, APIcast policies are created in the Lua programming language. So we need to setup an environment to do some Lua programming. Also, an actual APIcast server would be very nice to perform some local tests.
+
+Luckily APIcast makes it very easy to setup a development environment for APIcast using Docker and Docker Compose.
+
+### prerequisites
+This means both Docker and Docker compose must be installed.
+
+The version of Docker I currently use is:
```

Comment:
You can remove this line :point_up:

Reply:
addressed in https://github.com/3scale/APIcast/pull/1384